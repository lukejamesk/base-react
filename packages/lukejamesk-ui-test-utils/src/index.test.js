import testPackageExports from './testPackageExports'
import * as lukejameskTestUtils from '.'

testPackageExports('@lukejamesk/test-utils', lukejameskTestUtils)
