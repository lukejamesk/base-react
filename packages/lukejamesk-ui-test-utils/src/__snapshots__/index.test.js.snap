// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@lukejamesk/test-utils should match snapshot 1`] = `
Array [
  "changeWindowWidth",
  "createInteractor",
  "extendInteractor",
  "mergeInteractor",
  "setWindowWidth",
  "testPackageExports",
]
`;
